hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/RingbufferContainer.java

```diff
@@ -43,12 +49,11 @@
  * it is null to save space.
  */
 @SuppressWarnings("checkstyle:methodcount")
-public class RingbufferContainer implements IdentifiedDataSerializable {
+public class RingbufferContainer<T> implements IdentifiedDataSerializable, Notifier, Versioned {
```

Comment:
Yes, initially I had one big PR which I've decided to split into two. This change leaked from the other PR. Since I'm addressing the review comments I might as well move this change into the appropriate PR